install/src/Print.hs

```diff
@@ -0,0 +1,45 @@
+module Print where
+
+import           Development.Shake
+import           Development.Shake.Command
+import           Control.Monad.IO.Class
+import           Data.List                                ( dropWhileEnd
+                                                          , dropWhile
+                                                          )
+import           Data.Char                                ( isSpace )
+
+out :: MonadIO m => String -> m ()
+out  = liftIO . putStrLn
+
+out' :: MonadIO m => String -> m ()
+out' = out . ("    " ++)
```

Comment:
better naming since it has been promoted to a real util function? And/or documentation.